### ISSUE TITLE
find_peaks is not in the scipy

### ISSUE THREAD

**Issue by mahmood431226** [scipy/scipy#8827]
I was going to use find_peaks function in signal processing library in the scipy. so I import signal from scipy library but when I wanted to use this function, I faced to an error which said find_peaks is not in scipy.signal. then I upgrade scipy but this problem did not resolve

**Comment by WarrenWeckesser** [comment]
Verify that you actually upgraded to scipy version 1.1.0: `import scipy; print(scipy.__version__)`

**Comment by Dani-jay** [comment]
I am also having this problem, and have verified that I have version 1.1.0 of scipy. I am using Python 2.7 through sypder on anaconda cloud. The error that appears is: Attribute Error: 'module" object has no attribute 'signal'

**Comment by larsoner** [comment]
@Dani-jay it sounds like maybe you did only `import scipy`, you'll need to do `import scipy.signal` to import the `scipy.signal` namespace (doing `import scipy` is not enough to do it).

**Comment by RorisangSitoboli** [comment]
Use the form:

from scipy import signal
scipy.signal.correlate(A, B)

**Comment by ishipachev** [comment]
It's quite strange and "unpythonic" that it is not possible to write it as:

```
from scipy import signal
signal.correlate(A, B)
```

**Comment by RorisangSitoboli** [comment]
Definitely, @ishipachev it is strange. I ran into an error trying the method you pointed out. That would make sense if there was another module/library that also has a 'signal' function.

**Comment by larsoner** [comment]
> Definitely, @ishipachev it is strange. I ran into an error trying the method you pointed out.

I don't have any error doing what was suggested by @ishipachev:
```
>>> from scipy import signal
>>> signal.correlate
<function correlate at 0x7fe230916af0>
```
or with (in a separate Python instance, just to be safe) a variant of what @RorisangSitoboli said:
```
>>> import scipy
>>> from scipy import signal
>>> scipy.signal.correlate
<function correlate at 0x7fe230916af0>
```
I think that both of these should work for everyone with a functional/correct SciPy installation.

**Comment by larsoner** [comment]
> Definitely, @ishipachev it is strange. I ran into an error trying the method you pointed out.

I don't have any error doing what was suggested by @ishipachev:
```
>>> from scipy import signal
>>> signal.correlate
<function correlate at 0x7fe230916af0>
```
or with (in a separate Python instance, just to be safe) a variant of what @RorisangSitoboli said:
```
>>> import scipy
>>> from scipy import signal
>>> scipy.signal.correlate
<function correlate at 0x7fe230916af0>
```
I think that both of these should work for everyone with a functional/correct SciPy installation.

**Comment by ishipachev** [comment]
Hmm, I've tried to repeat it and wasn't able to run into the same problems. Sorry, seems like a possible false alarm.

**Comment by RorisangSitoboli** [comment]
Thanks, @larsoner. I will check my SciPy version. Maybe it is version dependent.
